planet/scripts/v1.py

```diff
@@ -711,8 +715,14 @@ def cancel_order(order_id, pretty):
 
 
 @click.option('--name', required=True)
-@click.option('--id', required=True,
-              help='One or more comma-separated item IDs')
+@click.option('--id', help='One or more comma-separated item IDs',
+                cls=RequiredUnless, this_opt_exists='ids_from_search')
+# Note: This is passed as a string, because --item-type is a required field for
+# both 'data search' and 'orders create'.
+@click.option('--ids_from_search',
+              help='Embedded data search')
+@click.option('--clip',
```

Comment:
it would be nice to be able to pass a geojson file path here.

Reply:
so there is currently a `--tools` option that requires a path to a file. the clip operation can be specified there already. the motivation for a string-based param is specifically to eliminate the need   to create extraneous files to run a command.

Reply:
... thinking about it more, I could see using the `curl` inspired `@file/path` string notation here to allow both, if you feel strongly it's beneficial

Reply:
oh I see, this is already the case for `--geom` under `data search`. I will add to be consistent

```
  GEOM

  Specify a geometry in GeoJSON format either as an inline value, stdin, or
  a file. '@-' specifies stdin and '@filename' specifies reading from a file
  named 'filename'.
```

thanks for bring this up 👍

Reply:
can `--clip @file` now, also updated docs